playbooks/common/openshift-cluster/evaluate_groups.yml

```diff
@@ -47,7 +47,7 @@
 
   # Use g_new_node_hosts if it exists otherwise g_node_hosts
   - set_fact:
-      g_node_hosts_to_config: "{{ g_new_node_hosts | default(g_node_hosts | default([])) }}"
+      g_node_hosts_to_config: "{{ g_new_node_hosts if g_new_node_hosts | length > 0 else g_node_hosts | default([]) }}"
```

Comment:
The following will do the same without the if/else (and also work when `g_node_hosts | length == 0`
```
g_node_hosts_to_config: "{{ g_new_node_hosts | default(g_node_hosts | default([], true), true) }}"
```